templates/resources/alert.md.tmpl

```diff
@@ -1,5 +1,10 @@
+<!---
```

Comment:
I guess technically these docs only said they were generated (initially), not auto-generated (every time), but that distinction was subtle and just generally confusing so I removed it in favor of this. Same with other files below.